samples/peripheral/lpuart/boards/nrf9160dk_nrf9160.overlay

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic */
```

Comment:
Nothing wrong on this line, but this is a nrf9160dk_nrf9160 overlay file, but based on the CMakeLists.txt file I would like to know if the nrf9160 is a whitelisted board or not ?

Reply:
removed CMakeLists.txt part